src/scss/shared/_brand.scss

```diff
@@ -33,21 +33,31 @@ $_o-forms-shared-brand-config: (
 		controls-negative-checked-background: oColorsByName('teal-30'),
 		toggle-inverse-selected: oColorsByName('teal-100'),
 		invalid-base: oColorsByName('crimson'),
+		invalid-base-inverse: oColorsByName('white'),
```

Comment:
"inverse" variables could be nested here to keep them together, and make it clear they're alternatives to other variables

```scss
@include oBrandDefine('o-forms', 'master', (
    'variables': map-merge($_o-forms-shared-brand-config, (
        invalid-base: 'some-color',
        error-summary-background: 'some-color',
        inverse: (
            invalid-base: 'some-other-color'
            error-summary-background: 'some-other-color'
        )
    )),
    'supports-variants': ()
));
```

```scss
background-color: _oFormsGet('error-summary-background', $from: 'inverse');
```
No need to change this though, it's more helpful when you have lots of themes :)

Reply:
That's good to know, thanks :-)